CommandHelper/LocalPackages/commands/chars/malice.yml

```diff
@@ -0,0 +1,120 @@
+  - libsdisguises.disguise.ocelot.-setinvisible
+  - libsdisguises.disguise.parrot.-setinvisible
+  - libsdisguises.disguise.pig.-setinvisible
+  - libsdisguises.disguise.rabbit.-setinvisible
+  - libsdisguises.disguise.sheep.-setinvisible
+  - libsdisguises.disguise.skeletonhorse.-setinvisible
+  - libsdisguises.disguise.squid.-setinvisible
+  - libsdisguises.disguise.villager.-setinvisible
+  - libsdisguises.disguise.cavespider.-setinvisible
+  - libsdisguises.disguise.enderman.-setinvisible
+  - libsdisguises.disguise.irongolem.-setinvisible
+  - libsdisguises.disguise.llama.-setinvisible
+  - libsdisguises.disguise.polarbear.-setinvisible
+  - libsdisguises.disguise.spider.-setinvisible
+  - libsdisguises.disguise.wolf.-setinvisible
+  - libsdisguises.disguise.pigzombie.-setinvisible
+  - libsdisguises.disguise.blaze.-setinvisible
+  - libsdisguises.disguise.creeper.-setinvisible
+  - libsdisguises.disguise.elderguardian.-setinvisible
+  - libsdisguises.disguise.endermite.-setinvisible
+  - libsdisguises.disguise.evoker.-setinvisible
+  - libsdisguises.disguise.ghast.-setinvisible
+  - libsdisguises.disguise.guardian.-setinvisible
+  - libsdisguises.disguise.husk.-setinvisible
+  - libsdisguises.disguise.magmacube.-setinvisible
+  - libsdisguises.disguise.shulker.-setinvisible
+  - libsdisguises.disguise.silverfish.-setinvisible
+  - libsdisguises.disguise.skeleton.-setinvisible
+  - libsdisguises.disguise.slime.-setinvisible
+  - libsdisguises.disguise.stray.-setinvisible
+  - libsdisguises.disguise.vex.-setinvisible
+  - libsdisguises.disguise.vindicator.-setinvisible
+  - libsdisguises.disguise.witch.-setinvisible
+  - libsdisguises.disguise.witherskeleton.-setinvisible
+  - libsdisguises.disguise.zombie.-setinvisible
+  - libsdisguises.disguise.zombievillager.-setinvisible
+  - libsdisguises.disguise.snowgolem.-setinvisible
+  - libsdisguises.disguise.enderdragon.-setinvisible
+  - libsdisguises.disguise.wither.-setinvisible
+  - libsdisguises.disguise.elderguardian.-setinvisible
+  - libsdisguises.disguise.giant.-setinvisible
+  - libsdisguises.disguise.illusioner.-setinvisible
+  - libsdisguises.disguise.killerbunny.-setinvisible
+  - libsdisguises.disguise.zombiehorse.-setinvisible
+  - libsdisguises.disguise.player
+  - ch.alias.underworld
+  - -essentials.protect.damage.fall
+  - ch.alias.tp
+  - essentials.fly
+  - essentials.speed.fly
+  - essentials.speed
+  - powers.tounderworld
+powers:
+- /effects
+- /items
+- /fly
+- /flyspeed <number>
+- /mindcontrol <player>
+- /stopmindcontrol
+- /underworld (travels to the UNDERWORLD!)
+- /disguise
+- /undisguise
+- /speak <player>
+- /confuse <player>
+- /carryplayer <player>
+- /dropplayer
+- /tp <player>
+- /jump 0-3
+- /run 0-3
+- /haste 0-4
+- /create <item> <quantity> <color>
+- /headdb
+run: 3
+speech: I cannot be defeated, don't try.
+trail: [
+  ]
+© 2018 GitHub, Inc.
+Terms
+Privacy
+Security
+Status
+Help
+Contact GitHub
+API
+Training
+Shop
+Blog
```

Comment:
Please remove all this random stuff at the bottom, you copied and pasted it by accident i think

Reply:
kk